content/parts/en/datasets.mdx

```diff
@@ -0,0 +1,55 @@
+---
+title: "Datasets"
+html_description: "Download datasets"
+related_datasets:
+  datasets:
```

Comment:
Why this double nesting? Couldn't we simply list the datasets under `related_datasets`, without the intermediary `datasets` variable? 🙂

Reply:
That's right, now that we've removed the title, this key is no longer necessary. Thanks!